nengo_gui/static/slider.js

```diff
@@ -145,6 +192,8 @@ Nengo.Slider.prototype.on_resize = function(width, height) {
     this.group.style.height = height - this.ax_top - 2 * this.border_size;
     this.group.style.marginTop = this.ax_top;
 
+    this.bound_labels.style.height = height - (this.ax_top - 10) - this.border_size ;
```

Comment:
It is unclear what the relevance of magic number `10` is in this line. This should be a named constant/variable. I wonder whether this height calculation is needed at all ...